src/components/dialogs/InvalidNrDialog.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-dialog v-model="dialog" persistent :attach="attach" max-width="60%">
+    <v-card class="notify-dialog">
+      <v-card-title>
+        <slot name="icon">
+          <v-icon large color="error">mdi-alert</v-icon>
+        </slot>
+        <span>
+          <slot name="title">Error Adding Name Request</slot>
+        </span>
+      </v-card-title>
+      <v-card-text>
+        <slot name="text">The specified name request is invalid or not found.</slot>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn large color="error" @click="emitClose" data-test="dialog-ok-button">OK</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
```

Comment:
Confirm with Scott but I think we should import the ContactInfo component here (ie, "if this error persists, please contact us" etc).

Reply:
Ok, i'm fine with that! There isn't actually a design for this work beyond 'reference the manage business' dash. So i'll keep this comment open and we can resolve this when we're able!